lib/jekyll/converter.rb

```diff
@@ -8,7 +8,9 @@ class Converter < Plugin
     #
     # Returns the String prefix.
     def self.highlighter_prefix(highlighter_prefix = nil)
-      @highlighter_prefix = highlighter_prefix if highlighter_prefix
+      if !defined?(@highlighter_prefix) || !highlighter_prefix.nil?
```

Comment:
@Crunch09 Absolutely! The change in behaviour is slight, but it is a change nevertheless that could cause problems. I think we'll go with it and revert in 3.2.1 if things go sour.